src/limits.cpp

```diff
@@ -554,6 +554,24 @@ void decay_object(ObjData *obj) {
     extract_obj(obj);
 }
 
+/* Proc poison */
+
+void sick_update(void) {
+    CharData *i, *next_char;
+    ObjData *j;
+
+    /* characters */
+    for (i = character_list; i; i = next_char) {
+        next_char = i->next;
+
+        if (GET_STANCE(i) >= STANCE_STUNNED) {
```

Comment:
Does this mean we don't want poison to kill people?  Why does stance matter at all?

Reply:
I have no earthly idea why this was written this way.  I just copy-pasted it from the function that previously used to call poison checks.